core/src/fetched-record-buffer.ts

```diff
@@ -1,20 +1,23 @@
 import { FetchedRecord } from './fetched-record'
 
+type PositionRecords = { [position: string]: FetchedRecord[] }
+type SubjectRecords = { [subject: string]: PositionRecords }
+
 export class FetchedRecordBuffer {
-  store: { [subject: string]: { [position: string]: FetchedRecord[] } }
+  store: SubjectRecords
 
   constructor() {
     this.store = {}
   }
 
-  static fromStore(store: { [subject: string]: { [position: string]: FetchedRecord[] } }) {
+  static fromStore(store: SubjectRecords) {
```

Comment:
`SubjectRecords` might not be the best name for this complex type. What do you think about reverting this? My preference is to PRY (please repeat yourself) over DRY (don't repeat yourself) in this case until this type is used somewhere externally or needed in more places

Reply:
This is updated